readme.md

```diff
@@ -54,7 +62,8 @@ console.log(results);
 
 Your Answer:
 ```text
-
+Find returns a collection.
```

Comment:
That's true, but the bigger issue is that `.find` is asynchronous, so `console.log(results)` would run before any Bobs had been found.